webextensions/api/sessions.json

```
@@ -204,7 +204,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/sessions/onChanged",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "36"
```

:x: https://github.com/chromium/chromium/commit/38f9258e83b4ce3e01f15c2e3009f43508892878 landed in [36.0.1918.0](https://github.com/chromium/chromium/releases/tag/36.0.1918.0) before 36.0.1985, but the parent feature was only promoted to stable in 37.

```suggestion
                "version_added": "37"
```